kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/NamespaceBundleOwnershipListenerImpl.java

```diff
@@ -59,7 +59,11 @@ public void onLoad(NamespaceBundle bundle) {
                     if (log.isDebugEnabled()) {
                         log.debug("[{}][{}] Trigger load callback for {}", brokerUrl, listener.name(), topic);
                     }
-                    listener.whenLoad(TopicName.get(topic));
+                    try {
+                        listener.whenLoad(TopicName.get(topic));
+                    } catch (Exception ex) {
+                        log.error("[{}][{}] Failed to do load for {}", brokerUrl, listener.name(), topic, ex);
+                    }
```

Comment:
We should handle the exception inside the `whenLoad` method. Or just catch the specific exception, e.g. if you worried about that `TopicName.get` might throw an exception.